### PR TITLE
Make AuthorizationVocabularyModel public and add MeasuresVocabularyModel 

### PR DESCRIPTION
**Pull request by chrisspre**
### Issues

*This pull request fixes issue #1940 

### Description

The two classes CapabilitiesVocabularyModel, AuthorizationVocabularyModel are marked internal.
This is inconsistent with CoreVocabularyModel and ValidationVocabularyModel.

Secondly, added Measures vocabulary for completeness. 

### Checklist (Uncheck if it is not completed)

- [x] *Test cases added*
- [x] *Build and test with one-click build and test script passed*

### CODE REVIEW

**Comment by xuzhg** [comment]
I'd like to see the related test cases covered. See https://github.com/OData/odata.net/tree/master/test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies

**Comment by chrisspre** [comment]
> I'd like to see the related test cases covered. See https://github.com/OData/odata.net/tree/master/test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies

@xuzhg Not sure I understand. There are already tests for Capabilities and Authorization. They run against the internal classes. 
What do you want me to do in addition? Are you just asking for Measures ?

File: test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/MeasuresVocabularyTests.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="AlternateKeysVocabularyTests.cs" company="Microsoft">
```
Comment (xuzhg) [inline test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/MeasuresVocabularyTests.cs:2]: >AlternateKeysVocabularyTests [](start = 20, length = 28)

update

File: test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/MeasuresVocabularyTests.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+using Xunit;
+
+namespace Microsoft.OData.Edm.Tests.Vocabularies
+{
+    public class MeasuresVocabularyModelTests
+    {
+
+        private readonly IEdmModel model = MeasuresVocabularyModel.Instance;
+
+
```
Comment (xuzhg) [inline test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/MeasuresVocabularyTests.cs:25]: remove unused line?

**Review by xuzhg — APPROVED** [review]
:shipit:

**Comment by xuzhg** [comment]
Just for new added.

---
In reply to: [741251481](https://github.com/OData/odata.net/pull/1939#issuecomment-741251481) [](ancestors = 741251481)

**Comment by xuzhg** [comment]
Would you please update the PR title?

And It's still in Draft?

File: test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/MeasuresVocabularyTests.cs (hunk 2 above)
Comment (chrisspre) [reply test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/MeasuresVocabularyTests.cs]: fixed

**Review by mikepizzo — APPROVED** [review]
:shipit:

**Pull request merged**